### PR TITLE
Ledger test: path

### PR DESCRIPTION
**Pull request by tock-ibm**

### CODE REVIEW

File: test/queries/ledger_test.go [hunk 1]
```diff
@@ -186,3 +192,187 @@ func TestLedgerBlockQueries(t *testing.T) {
...
+			name:            "1 to 10",
+			start:           1,
+			end:             10,
+			expectZeroBased: []uint64{9, 8, 0},
+		},
+		{
+			name:            "5 to 15",
+			start:           5,
+			end:             15,
+			expectZeroBased: []uint64{14, 12, 8, 4},
```
Comment (cendhu) [inline test/queries/ledger_test.go:260]: Using different expected values creates confusion. In the unit-test, we pass non-zero based value, i.e., 15, 13, 8, 5 but here we pass zero based and then add 1 in the assertion. Initially, I was wondering why block list is wrong and realised after looking at the `+1`. You can have the zero based value in the comment and actual value in the test.

File: test/queries/ledger_test.go [hunk 2]
```diff
@@ -186,3 +192,187 @@ func TestLedgerBlockQueries(t *testing.T) {
...
+			name:            "1 to 1",
+			start:           1,
+			end:             1,
+			expectZeroBased: []uint64{0},
+		},
+		{
+			name:            "6 to 6",
+			start:           6,
+			end:             6,
+			expectZeroBased: []uint64{5},
```
Comment (cendhu) [inline test/queries/ledger_test.go:278]: having 6 instead of 5 would make readability easier

Comment (tock-ibm) [reply test/queries/ledger_test.go]: done

File: test/queries/ledger_test.go (hunk 1 above)
Comment (tock-ibm) [reply test/queries/ledger_test.go]: done

**Pull request merged**